app/src/main/res/layout/login_realtivelayout.xml

```diff
@@ -7,134 +7,137 @@
     android:paddingTop="16dp"
     android:paddingRight="10dp">
 
-        <TextView
-            android:id="@+id/Back"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="&lt; Back"
-            android:textColor="@android:color/black"
-            android:textSize="20dp"
-            android:textStyle="bold" />
+    <TextView
+        android:id="@+id/textViewBack"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="&lt; Back"
+        android:textColor="@android:color/black"
+        android:textSize="20dp"
+        android:textStyle="bold" />
 
-    <RelativeLayout
+    <ImageView
+        android:id="@+id/imageViewHrLogo"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/textViewBack"
+        android:layout_marginTop="30dp"
+        android:layout_marginBottom="30dp"
+        android:paddingLeft="10dp"
+        android:src="@drawable/hrlogo" />
+
+    <TextView
+        android:id="@+id/textViewProceedWithYour"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignBaseline="@id/Back"
-        android:paddingLeft="10dp">
+        android:layout_below="@+id/imageViewHrLogo"
+        android:paddingLeft="10dp"
+        android:text="Proceed with your"
+        android:textColor="@android:color/black"
+        android:textSize="24dp" />
+
+    <TextView
+        android:id="@+id/textViewLogin"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/textViewProceedWithYour"
+        android:layout_marginBottom="50dp"
+        android:paddingLeft="10dp"
+        android:text="Login"
+        android:textColor="@android:color/black"
+        android:textSize="24dp"
+        android:textStyle="bold" />
```

Comment:
I didn't recommend camel case to you to set id value but snake_case. 
Please make id style to snake_case 
ref : https://github.com/kangraemin/AndroidStudy/pull/11#discussion_r456919183

Reply:
Fix  ids in relative layout

commit id is [092e7ac]